### DISCUSSION TITLE
Replacing previous notifications

### DISCUSSION THREAD

**Discussion by grtcdr** [hoodie/notify-rust#146] [Q&A] [answered]
Hi :smile:

Is there a way to replace an active notification?

[notify-send.py](https://github.com/phuhl/notify-send.py#about) is a helper tool which utilizes a feature called `replaces-id` that can be used to replace an active notification, can we do something similar with `notify-rust`?

**Comment by hoodie** [comment, marked as answer]
Hi there, yes there is.
Several ways actually.
I hope the [update example](https://github.com/hoodie/notify-rust/blob/main/examples/update.rs) illustrates them.

**Reply by grtcdr** [reply]
Hi, thanks a lot! I'll check them out :)
